src/components/Home/DescriptionBoxes.tsx

```diff
@@ -0,0 +1,43 @@
+// Edit span tags to white after bg change to black
+
+export default function DescriptionBoxes() {
```

Comment:
For consistency change to:
`function DescriptionBoxes(){...}`
`export default DescriptionBoxes;`